libs/rudder-integration-facebook-react-native/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@rudderstack/rudder-integration-facebook-react-native",
+  "title": "Rudder Integration Facebook React Native",
```

Comment:
```suggestion
  "title": "RudderStack Facebook React Native Integration",
```

Reply:
Also, should we start calling it `Meta` instead of `Facebook`?
CC @ChryssaAliferi